lib/client-api/src/client_api.js

```diff
@@ -226,9 +226,9 @@ export default class ClientApi {
     this._storyStore.getStoryKinds().map(kind => {
       const fileName = this._storyStore.getStoryFileName(kind);
 
-      const stories = this._storyStore.getStories(kind).map(name => {
-        const render = this._storyStore.getStoryWithContext(kind, name);
-        return { name, render };
+      const stories = this._storyStore.getStories(kind).map(story => {
+        const render = this._storyStore.getStoryWithContext(kind, story.name);
+        return { name: story.name, render, parameters: story.parameters };
```

Comment:
You can destructure `name` and `parameters` out of `story` or spread the `story` object in the return value right?

Reply:
@Keraito I'm not sure how that's done.  Could you provide an example?

Reply:
Either 
1. `const stories = this._storyStore.getStories(kind).map(({name, parameters}) => {` and then you don't have to do `story.name`/`story.parameters` and just use `name` and `parameters` instead, which means your return statement becomes `return { name, render, parameters }`.
2. or your return statement becomes `return { render, ...story }` and it will insert the `name` and `parameters` properties automatically in the return object. 

If the `story` contained more properties, those would be added as well to the return object (which is not too great), but in this case it only contains name and parameters anyways. Either of the above options is fine tho, pick whichever you like. Docs on `...`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

Reply:
Have gone with option 1, thanks.